README.md

```diff
@@ -52,9 +54,15 @@ OGame-clone browser based game engine.
 1. Move source files of the project to your webserver's directory.
 1. Install PHP dependencies.
     - ``composer install --no-dev``
-1. Run installation wizard: http://your_server_address:port/install
+1. Run installation wizard: ``http(s)://<your_server_address>:<port>/install``
 1. Remove ``install/`` directory
 
+### Docker
+
+1. Prepare config file ``mkdir docker && cp config.php docker && cp includes/constants.php docker && chown 33:33 docker/*.php``
+1. Build and run services ``docker-compose up -d --build``
+1. Run installation wizard: ``http(s)://<your_server_address>:<port>/install``
```

Comment:
It might be a good idea to add a note about the mysql's server host, username & password, to be used during the installation, since the compose does start that server as well. It probably pretty obvious for people how know docker and have read the compose file, but still worth documenting for people not so accustomed with that environment.

Reply:
Like the web port configuration, I extracted these configurations into environment variables, and there are clear steps in the readme to ask users to check and modify these configurations, and also point out the value of the database host